sdks/python/apache_beam/yaml/yaml_udf_test.py

```diff
@@ -0,0 +1,338 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import io
+import logging
+import os
+import shutil
+import tempfile
+import unittest
+from unittest import mock
+
+import apache_beam as beam
+from apache_beam.io import localfilesystem
+from apache_beam.io.filesystem import CompressedFile, CompressionTypes
+from apache_beam.io.gcp import gcsio, gcsfilesystem
+from apache_beam.options import pipeline_options
+from apache_beam.testing.util import equal_to, assert_that
+from apache_beam.yaml.yaml_transform import YamlTransform
+
+MOCK_UDF_FILE_PREFIX = "/path/to/udf"
```

Comment:
Remove. 

In fact, I think you can remove all the mocks in this file (e.g. no need to explicitly test gcs, just trust that Filesystems has its own tests).